--- origsrc/taglib-1.5/Makefile.am	2008-01-31 12:18:39.000000000 -0600
+++ src/taglib-1.5/Makefile.am	2008-03-23 20:41:54.375000000 -0500
@@ -45,9 +45,9 @@
 	echo "GENERATE_HTML       = YES" >> taglib.doxyfile ; \
 	echo "GENERATE_MAN        = NO" >> taglib.doxyfile ; \
 	echo "GENERATE_LATEX      = NO" >> taglib.doxyfile ; \
-	echo "HTML_HEADER         = doc/common/header.html" >> taglib.doxyfile ; \
-	echo "HTML_FOOTER         = doc/common/footer.html" >> taglib.doxyfile ; \
-	echo "HTML_STYLESHEET     = doc/common/doxygen.css" >> taglib.doxyfile ; \
+	echo "HTML_HEADER         = $(top_srcdir)/doc/api-header.html" >> taglib.doxyfile ; \
+	echo "HTML_FOOTER         = $(top_srcdir)/doc/api-footer.html" >> taglib.doxyfile ; \
+	echo "HTML_STYLESHEET     = $(top_srcdir)/doc/taglib-api.css" >> taglib.doxyfile ; \
 	echo "FILE_PATTERNS       = *.h" >> taglib.doxyfile ; \
 	echo "PREDEFINED          = DO_NOT_DOCUMENT DOXYGEN" >> taglib.doxyfile ; \
 	echo "EXTRACT_ALL         = YES" >> taglib.doxyfile ; \
