--- origsrc/taglib-1.9.1/taglib/toolkit/trefcounter.h	2013-10-08 10:50:01.000000000 -0500
+++ src/taglib-1.9.1/taglib/toolkit/trefcounter.h	2014-12-05 12:41:29.675151900 -0600
@@ -32,7 +32,7 @@
 #ifdef __APPLE__
 #  include <libkern/OSAtomic.h>
 #  define TAGLIB_ATOMIC_MAC
-#elif defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__CYGWIN__)
+#elif defined(WIN32) || defined(_WIN32) || defined(__WIN32__)
 #  define NOMINMAX
 #  include <windows.h>
 #  define TAGLIB_ATOMIC_WIN
